tests/WebAppCallsWebApiCallsGraph/TodoListService/TodoListService.csproj

```diff
@@ -16,9 +16,9 @@
     <ProjectReference Include="..\..\..\src\Microsoft.Identity.Web.MicrosoftGraph\Microsoft.Identity.Web.MicrosoftGraph.csproj" />
     <ProjectReference Include="..\..\..\src\Microsoft.Identity.Web\Microsoft.Identity.Web.csproj" />
     <PackageReference Include="Microsoft.Extensions.Caching.StackExchangeRedis" Version="5.0.1" />
-    <PackageReference Include="Microsoft.IdentityModel.Tokens" Version="6.10.0" />
```

Comment:
This change could have stayed.

Reply:
i know, but since it's just the dev app, i just reverted it back as well.